feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt

```diff
@@ -99,6 +100,10 @@ fun SearchRoot(
 
     val keyboardController = LocalSoftwareKeyboardController.current
 
+    if (categoryId != null) {
+        viewModel.onCategorySelected(categoryId)
```

Comment:
I think we shouldn't call viewModel moethod in Composable function directly. 👀 
So how about using LaunchedEffect?

LaunchedEffect(initialCategoryId) {
  viewModel.onCategorySelected(categoryId)
}